Description: Fix Latex code that made Dokygen halt the build
Author: Gert Wollny <gw.fossdev@gmail.com> 
Last-Update: 2016-02-02
Forwardedd: no
--- a/Common/Core/vtkAbstractArray.h
+++ b/Common/Core/vtkAbstractArray.h
@@ -384,7 +384,7 @@
   // prominence P, we sample N values, with N = f(T; P, U).
   // We want f to be sublinear in T in order to interactively handle large
   // arrays; in practice, we can make f independent of T:
-  // \f$ N >= \frac{5}{P}\mathrm{ln}\left(\frac{1}{PU}) \f$,
+  // \f$ N >= \frac{5}{P}\mathrm{ln}\left(\frac{1}{PU}\right) \f$,
   // but note that small values of P are costly to achieve.
   // The default parameters will locate prominent values that occur at least
   // 1 out of every 1000 samples with a confidence of 0.999999 (= 1 - 1e6).
--- a/Common/Core/vtkMath.h
+++ b/Common/Core/vtkMath.h
@@ -661,14 +661,14 @@
   // The output is provided by overwriting the input A with a matrix of the same size as
   // A containing all of the information about L and U. If the output matrix is
   // \f$ A* = \left( \begin{array}{cc}
-  // a & b \\ %
+  // a & b \\
   // c & d \end{array} \right)\f$
   // then L and U can be obtained as:
   // \f$ L = \left( \begin{array}{cc}
-  // 1 & 0 \\ %
+  // 1 & 0 \\
   // c & 1 \end{array} \right)\f$
   // \f$ U = \left( \begin{array}{cc}
-  // a & b \\ %
+  // a & b \\
   // 0 & d \end{array} \right)\f$
   //
   // That is, the diagonal of the resulting A* is the diagonal of U. The upper right
